File: spring-boot-VMM/src/main/java/pp/project/vmm/config/security/rest/response/UserInfo.java
package pp.project.vmm.config.security.rest.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserInfo {

    private UUID userId;

    private String username;

    private String firstName;

    private String lastName;

    private String phoneNumber;
}

File: angular-VMM/src/app/models/Employee/UserInfo.ts
export class UserInfo{
    userId: string;
    username: string;
    firstName: string
    lastName: string;
    phoneNumber: string;


    constructor(userId: string, username: string, firstName: string, lastName: string, phoneNumber: string) {
        this.userId = userId;
        this.username = username;
        this.firstName = firstName;
        this.lastName = lastName;
        this.phoneNumber = phoneNumber;
    }
}
